src/style.css

```
@@ -1 +1,29 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url('./images/star.svg');
+  margin-right: 4px;
+  background-position: center;
+  background-repeat:no-repeat;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 :nth-child(-n+1),
+.stars--2 :nth-child(-n+2),
+.stars--3 :nth-child(-n+3),
+.stars--4 :nth-child(-n+4),
+.stars--5 :nth-child(-n+5){
+    background-image: url('./images/star-active.svg'); /* Фонове зображення активної зірки */
+}
```

Comment is redundant here
```suggestion
.stars--5 :nth-child(-n+5){
  background-image: url('./images/star-active.svg');
```